### ISSUE TITLE
Missing Empowered Spells Support

### ISSUE THREAD

**Issue by Tuller** [tullamods/Dominos#674]
Dominos Cast does not support empowered spells at the moment (I've not had access to the beta to test this feature).

**Comment by Tuller** [comment]
The castbar now shows up when casting an empowered spell, but does not yet indicate the level reached yet. I think, simply, I think I'll be starting with a simpler UI for it (just tick marks on the current bar).

**Comment by Leuchtturm123** [comment]
> The castbar now shows up when casting an empowered spell, but does not yet indicate the level reached yet. I think, simply, I think I'll be starting with a simpler UI for it (just tick marks on the current bar).

Yes this is working now. Cool would be to clearly see the different empowered level like the original bar does but I am looking foreward with your idea of a simpler approach.

**Comment by vexir** [comment]
@Tuller Would it be possible to turn off Dominos control/skinning of the cast bar for now? Or if there is a way to do that in the current add-on that would be great to know.
